bigbluebutton-html5/client/compatibility/kurento-extension.js

```diff
@@ -29,12 +27,29 @@ Kurento = function (
   this.webRtcPeer = null;
   this.mediaCallback = null;
 
-  this.voiceBridge = `${voiceBridge}-SCREENSHARE`;
+  this.renderTag = tag;
+  this.voiceBridge = voiceBridge;
+  this.userId = userId;
   this.internalMeetingId = internalMeetingId;
 
-  this.userId = userId;
-  this.userName = userName;
-  this.caleeName = caleeName;
+  // Optional parameters are: userName, caleeName, chromeExtension, wsUrl, iceServers,
+  // chromeScreenshareSources, firefoxScreenshareSource
+
+  Object.assign(this, options);
+
+  if (this.wsUrl == null) {
+    this.defaultPath = 'bbb-webrtc-sfu';
```

Comment:
Shouldn't we point to https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-html5/private/config/settings-development.json#L74 via `Meteor.settings....` ?

Reply:
The config is passed through in the screenshare/audio bridge via the `options` parameter. I didn't want to explicitly fetch the Meteor setting in that plain .js script, and the check is just there because the parameter is optional and it's a fallback to the default url (hostname/bbb-webrtc-sfu).

Reply:
e.g.: https://github.com/bigbluebutton/bigbluebutton/pull/5842/files#diff-a110fc2d4f118e42b5d53e136f39f8fcR35